dist/s_all

```diff
@@ -47,6 +47,12 @@ errchk()
 		return
 	fi
 
+	# Return if evergreen validate runs sucessfully.
+	if echo "$1" | grep -q evergreen && [ "$(cat $2)" = "../test/evergreen.yml is valid" ] ; then
```

Comment:
Is $2 a temporary file created?

Reply:
Yes, the $2 file is temporary. We create the file in `run()` and remove it in `errchk()`.